exactonline/http.py

```diff
@@ -290,6 +293,10 @@ def _http_request(url, method=None, data=None, opt=None):
     else:
         opener = request.build_opener()
 
+    logger.debug(
+        'Outgoing request for {url} '
+        'using method{method}'.format(url=url, method=method)
+    )
```

Comment:
Please use:
```
logger.debug(
    'Outgoing request for {url} using method {method}'
    .format(url=url, method=method))
```
or
```
logger.debug(
    'Outgoing request for {url} using method {method}'.format(
        url=url, method=method))
```
Needlessly breaking up a string makes it harder to grep for.

Also:
`method {method}` (with space).

Reply:
I tend to use [`black`](https://black.readthedocs.io/en/stable/) for formatting, so wasn't sure what was the best way to split. Adapted it as per your suggestion.